app/server/syntax/file_map/nodes_find.go

```diff
@@ -99,6 +99,45 @@ func isIncludeAndContinueNode(node Node) bool {
 	return !config.ignore
 }
 
+func isUnwrapNode(node Node) bool {
+	setNodeType(&node)
+	config := unwrapNodeMap.getConfig(node.Type, node.Lang)
+
+	if config == nil {
+		return false
+	}
+
+	return true
+}
+
+func unwrapNodeIndex(node Node) *int {
+	idx := new(int)
+
+	if isUnwrapNode(node) {
+
+		if node.Lang == shared.LanguageTypescript || node.Lang == shared.LanguageTsx {
+			switch node.TsNode.ChildCount() {
```

Comment:
I don't love this, and it's probably a bit brittle, though it seems to work fine on all the test cases I've thought of.